src/Pipeline.cpp

```diff
@@ -808,6 +825,11 @@ void Pipeline::prepare_jit_call_arguments(RealizationArg &outputs, const Target
                                           bool is_bounds_inference, JITCallArgs &args_result) {
     user_assert(defined()) << "Can't realize an undefined Pipeline\n";
 
+    FindVariablesWithUnsetParams find_unset;
```

Comment:
I think this code is a hot path, so we can't do IR traversal here.

Reply:
Blarg. That's no good. I'm open for better suggestions.